data-science/data_science_intern_challenge.md

```diff
@@ -0,0 +1,43 @@
+# Guidebook Data Science Intern Challenge
+
+Congratulations on making it to this stage of the interview process! This exercise will test your python coding skills, your data analysis skills, and give you a sense of the kinds of work you would do at Guidebook as a Data Science Intern. In this challenge, you'll first demonstrate your python abilities and perform some specified processing on raw data to produce a dataset. Then, you'll take a deeper look into the dataset you create in part 1 and write up some of your findings.
+
+Please complete all parts to the best of your ability. This should take 1-2 hours to complete.
+
+
+## Part 1: User Histograms
+
+Often at Guidebook, we aggregate metrics events associated to a user or a guide as a way of describing that entity.
+For the first part of this exercise you'll do some of this to demonstrate your Python coding abilities.
+Specifically, you are going to write a code which given a list of metrics event dictionaries of form:
```

Comment:
Small grammar question / comment

```suggestion
Specifically, you are going to write code which given a list of metrics event dictionaries of form:
```

I think there's an extra "a" in here that's not needed/wanted.